sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContainer.md

```diff
@@ -0,0 +1,203 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
```

Comment:
please fill the online version metadata

Reply:
should we just assume the link that it will live on based on the file structure?